go.mod

```diff
@@ -21,8 +21,9 @@ require (
 	k8s.io/kubelet v0.22.1
 )
 
+exclude cloud.google.com/go/compute v1.7.0
```

Comment:
Can we exclude the old version cloud.google.com/go v0.65.0?

Reply:
It will choose then even older version of this package - v0.54.0